card/config/initializers/02_patches/active_record.rb

```diff
@@ -0,0 +1,66 @@
+# pluck_in_batches:
```

Comment:
Ah, makes sense and good point about potential use. Recently ran into the fact that Rails3 doesn't support multi-column pluck, bummer.